src/encoded/schemas/experiment_set.json

```diff
@@ -97,6 +97,13 @@
             "lookup": 20,
             "formInput": "textarea"
         },
+        "disease_name": {
+            "title": "Disease Name",
+            "description": "Ontology term for the disease name associated with the set.",
+            "type":  "string",
```

Comment:
I'd suggest making this property an array field so more than one disease/disorder term can be added.